tasks/page/processBOM.js

```diff
@@ -49,10 +49,16 @@ if (require.main !== module) {
     }
   }, '')
 
-  const file = fs.readFileSync(yamlPath)
-  if (file != null) {
-    kitspaceYaml = yaml.safeLoad(file)
+  try {
+    const file = fs.readFileSync(yamlPath)
+
+    if (file != null) {
+      kitspaceYaml = yaml.safeLoad(file)
+    }
+  } catch (error) {
+    console.log('WARNING: NO KITSPACE YAML FILE')
```

Comment:
We actually allow no yaml file, if people use the default file locations.

Reply:
I think rather than a try/catch it would be better to check if the file is valid.

Reply:
Updated :+1: